mep3_navigation/include/mep3_navigation/stuck_behavior/stuck_behavior.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef MEP3_NAVIGATION__STUCK_BEHAVIOR_HPP_
+#define MEP3_NAVIGATION__STUCK_BEHAVIOR_HPP_
+
+#include <chrono>
+#include <memory>
+#include <utility>
+
+#include "nav2_behaviors/timed_behavior.hpp"
+#include "mep3_msgs/action/move.hpp"
+#include "mep3_msgs/msg/motion_properties.hpp"
+#include "nav2_util/node_utils.hpp"
+#include "ruckig/ruckig.hpp"
+#include "mep3_navigation/stuck_detector.hpp"
+
+namespace mep3_navigation
+{
+  using ActionT = mep3_msgs::action::Move;
+
+  class StuckBehavior : public nav2_behaviors::TimedBehavior<ActionT>
+  {
+  public:
+    StuckBehavior(double linear_x=0.4, double angular_z=0.4);
+    ~StuckBehavior();
+
+    virtual nav2_behaviors::Status onRun(const std::shared_ptr<const typename ActionT::Goal> command) final override;
+
+    virtual nav2_behaviors::Status onCycleUpdate() final override;
+
+  protected:
+    void initializeTranslation(double diff_x, double diff_y);
+    void regulateTranslation(geometry_msgs::msg::Twist *cmd_vel, double diff_x, double diff_y);
```

Comment:
```suggestion
    void initializeTranslation(double diff_x, double diff_y);
    void regulateTranslation(geometry_msgs::msg::Twist *cmd_vel, double diff_x, double diff_y);
```
Cini mi se da se ove dvije ne koriste